### PR TITLE
Add workflow for filing live demo issues

### LINKED ISSUE #10: Make GHA for filing workshop issues
Rather than directly opening issues needed for workshop live demos, let's create a GHA to automagically file live demo issues, which can live in `live-demo-issues/...markdown files...`. This has a couple benefits:

- More concrete chance to have those issues reviewed
- Helps out with #9 - if we use the template again, much easier to get issues up and running

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #10

This PR adds a manually-triggered workflow to file issues in [`issue-templates`](https://github.com/AlexsLemonade/2023-chop-training-demo/tree/695a8829275a6253c89f348ad4800a6ad33453a7/.live-demo-content/issue-templates).

I also include two inputs:
- First, the github username for who will be performing the live demos. This username gets assigned all issues except the code review issue. For this workshop, we'd provide `sjspielman` here. 
- Second, the github username for who will be filing the issue for code review, and that username gets assigned accordingly. For this workshop, we'd provide `jaclyn-taroni` here.

Note also that using a matrix here for the 4 working with branches issues is wayyyyy more trouble than it's worth; we'd need a separate job (even steps that don't use a matrix variable get run once per matrix value!), and issue titles are not the same among those 4. So, individual steps it is.

### CODE REVIEW

File: .github/workflows/file-live-demo-issues.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+on:
+  workflow_dispatch:
+    inputs:
+      demo_lead_username:
+        description: The github username of who will be performing the live demonstrations
+        required: true
+      demo_support_username:
+        description: The github username of who will file the PR to support the live code demonstration
+        required: true
```
Comment (jaclyn-taroni) [inline .github/workflows/file-live-demo-issues.yml]: Assuming that material will get split up the same way strikes me as too strong of an assumption.

**Review by jaclyn-taroni — COMMENTED** [review]
In my opinion, it's too strong an assumption that issues would get divided up the same way every time. So I'm returning this so you can test setting `required: false` to both of these inputs. (I'd test using one issue for each role to reduce the amount of spam.

**Comment by sjspielman** [comment]
Tested and issues were filed! Maybe we just leave those open and assign manually?

**Comment by jaclyn-taroni** [comment]
Can you add docs to the main README, please? Thinking we'll appreciate that when we go to templatize (#9).

**Comment by sjspielman** [comment]
I updated the README to explain setup in https://github.com/AlexsLemonade/2023-chop-training-demo/pull/17/commits/aa49d572229555755368d74018a90046156de1ab, but I left some placeholder text in italics that I figure can be filled in when the PR text as part of https://github.com/AlexsLemonade/2023-chop-training-demo/issues/5 is filed.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+If you do not provide these inputs, you will have to manually assign the resulting issues.
+  * The `demo_lead_username` input should take the username to assign to all issues _except_ [the issue associated with demonstrating PR review](.live-demo-content/issue-templates/performing-code-review.md).
+  * The `demo_support_username` input should take the username to assign _only_ to [the issue associated with demonstrating PR review](.live-demo-content/issue-templates/performing-code-review.md).
+
+### Filing a PR for live code review
+
+* Whoever is assigned [the issue associated with demonstrating PR review](.live-demo-content/issue-templates/performing-code-review.md) will need to prepare a branch and file a PR, with several commits (whose messages have varying levels of information), that will be reviewed during the live demonstration.
+* This branch should ultimately modify contents of [this baseline notebook](analyses/explore-spotify-variation.Rmd) to match [this updated notebook](.live-demo-content/explore-spotify-variation-with-UMAPs.Rmd).
+* A PR should then be filed to `main`, using _this forthcoming file which contains the PR text._
```
Comment (jaclyn-taroni) [inline README.md]:
```suggestion
* A PR should then be filed to `main`, using _TODO: this forthcoming file which contains the PR comment text._
```

Command+F "TODO" is sometimes helpful.

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻

**Pull request merged**